Fix `rule-empty-line-before` false positives for shared-line comments with `except: ["after-single-line-comment"]` (#9394)
stylelint/stylelint: .changeset/olive-pens-yawn.md

---
"stylelint": patch
---

> Fixed: `rule-empty-line-before` false positives for `except: ["after-single-line-comment"]` when the comment and the rule share a line

Reviewer: ```suggestion
Fixed: `rule-empty-line-before` false positives for shared-line comments with `except: ["after-single-line-comment"]`
```
